config/cloud.cfg.tmpl

```diff
@@ -349,4 +349,8 @@ system_info:
 {% elif variant in ["dragonfly"] %}
    network:
       renderers: ['freebsd']
+{% elif variant in ["rhel", "fedora"] %}
+   network:
+      renderers: ['network-manager', 'netplan', 'eni', 'sysconfig', 'freebsd', 'netbsd',
```

Comment:
For RHEL/Fedora, please use `['netplan', 'network-manager', 'networkd', 'sysconfig', 'eni']`

Reply:
ok